scripts/zones/Castle_Zvahl_Baileys/mobs/Marquis_Naberius.lua

```diff
@@ -18,6 +34,7 @@ entity.onMobDeath = function(mob, player, optParams)
 end
 
 entity.onMobDespawn = function(mob)
+    xi.mob.updateNMSpawnPoint(mob, spawnPoints)
```

Comment:
not really a point to use this function if only 1 spawn point listed, besides consistancy?

Reply:
Oops, I forgot to generate more spawn points for that guy. I'll get to that tonight.

Reply:
Got some more spawn points made for this NM